recipes/fabio/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.3.0" %}
+
+package:
+    name: fabio
+    version: {{ version }}
+
+source:
+    url: https://github.com/kif/fabio/archive/v{{ version }}.tar.gz
+    fn: fabio-v{{ version }}.tar.gz
+    sha256: ec4586a676dab02931bed2f1e0ef227c43259b4f60c0cfb603c30e02876164e8
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy x.x
+        - cython
+        # could use opencl, but need the package on conda-forge first
+        # - opencl
+
+    run:
+        - python
+        - numpy x.x
+
+test:
+    imports:
+        - fabio
+        - fabio.test
+        - fabio.third_party
+
+about:
+    home: http://fable.wiki.sourceforge.net/fabio
+    license: GNU General Public License v2 or later (GPLv2+)
```

Comment:
Here is the [`copyright`]( https://github.com/kif/fabio/blob/v0.3.0/copyright ) file for reference.